`lib/bigdecimal.rb` is missing from `s.files` in the `gemspec`

Which will yield,

```
make "DESTDIR="
make: *** No rule to make target '../../lib/bigdecimal.rb', needed by 'Makefile'.  Stop.

make failed, exit code 2
```

Same issue here

```
Updating installed gems
Updating bigdecimal
Building native extensions. This could take a while...
ERROR:  Error installing bigdecimal:
        ERROR: Failed to build gem native extension.

    current directory: /usr/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.0/ext/bigdecimal
/usr/bin/ruby -r ./siteconf20181223-21678-1uimf7d.rb extconf.rb
checking for labs() in stdlib.h... yes
checking for llabs() in stdlib.h... yes
checking for finite() in math.h... yes
checking for isfinite() in math.h... no
checking for struct RRational in ruby.h... no
checking for rb_rational_num() in ruby.h... yes
checking for rb_rational_den() in ruby.h... yes
checking for rb_array_const_ptr() in ruby.h... yes
checking for rb_sym2str() in ruby.h... yes
creating Makefile

current directory: /usr/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.0/ext/bigdecimal
make "DESTDIR=" clean
make: *** No rule to make target '../../lib/bigdecimal.rb', needed by 'Makefile'.  Stop.

current directory: /usr/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.0/ext/bigdecimal
make "DESTDIR="
make: *** No rule to make target '../../lib/bigdecimal.rb', needed by 'Makefile'.  Stop.

make failed, exit code 2

Gem files will remain installed in /usr/lib/ruby/gems/2.5.0/gems/bigdecimal-1.4.0 for inspection.
Results logged to /usr/lib/ruby/gems/2.5.0/extensions/x86_64-linux/2.5.0/bigdecimal-1.4.0/gem_make.out
Gems updated: bigdecimal
```

same problem.
#### version:
ruby 2.4.1
rails 5.2.1

#### error

make: *** No rule to make target `../../lib/bigdecimal.rb', needed by
`Makefile'.  Stop

--- a/bigdecimal.gemspec
+++ b/bigdecimal.gemspec
@@ -24,6 +24,7 @@ Gem::Specification.new do |s|
     ext/bigdecimal/extconf.rb
     ext/bigdecimal/util/extconf.rb
     ext/bigdecimal/util/util.c
+    lib/bigdecimal.rb
     lib/bigdecimal/jacobian.rb
     lib/bigdecimal/ludcmp.rb
     lib/bigdecimal/math.rb

